_docs/_latest/configure/notifier/index.md

```diff
@@ -254,19 +277,21 @@ process below:
 1. [Sign up for a SendGrid account](https://sendgrid.com/).
 1. Create a general
     [API Key](https://sendgrid.com/docs/User_Guide/Settings/api_keys.html).
-1. Edit the following in `forseti_conf_server.yaml`:
-    1. `email_recipient`
-       * **Description**: The Email address of notification recipient.
-    1. `email_sender`
-       * **Description**: The Sender email address for notifications
-    1. `sendgrid_api_key`
-       * **Description**: The API key for SendGrid email service.
+1. Edit the `email_connector` section in `forseti_conf_server.yaml` to provide
+SendGrid specific details.
 
 Note that SendGrid automatically includes an invisible tracking pixel in your
 emails. This may cause email warnings about opening images. To disable this,
 disable SendGrid
 [Open Tracking](https://sendgrid.com/docs/User_Guide/Settings/tracking.html#-Open-Tracking).
 
+### Add the email connector
+
+To add the email connector of your choice, create the connector specific class 
+under `google.cloud.forseti.common.util.email` that inherits the base email 
+connector class, base_email_connector.py, and implements the methods to send out 
```

Comment:
I think this section needs to have more information

Reply:
Ok